### PR TITLE
Anaconda WebUI workflow logging improvement

### PR DESCRIPTION
**Pull request by velezd**

### CODE REVIEW

File: libpermian/plugins/anaconda_webui/__init__.py [hunk 1]
```diff
@@ -447,10 +448,25 @@ def _wait_for_webui(self):
...
             else:
                 break
 
+    def _take_vm_screenshot(self, name):
+        """ Creates screenshot of VMs screeen
+
+        :param name: screenshot filename
+        :type name: str
+        """
+        screenshot_file = os.path.join(
```
Comment (pholica) [inline libpermian/plugins/anaconda_webui/__init__.py]: please use temporary file for the screenshot and then just call `self.addLog` with `copy_file=True`. That way, you don't need to deal with the file paths in any way.

**Review by pholica — CHANGES_REQUESTED** [review]

File: libpermian/plugins/anaconda_webui/__init__.py (hunk 1 above)
Comment (velezd) [reply libpermian/plugins/anaconda_webui/__init__.py]: I was assuming virsh wouldn't like it, thanks

**Review by pholica — DISMISSED** [review]

File: libpermian/plugins/anaconda_webui/__init__.py [hunk 2]
```diff
@@ -447,10 +448,21 @@ def _wait_for_webui(self):
                     # Connection refused - webui not yet started
                     continue
                 else:
+                    self._take_vm_screenshot('screenshot_webui_not_accessible.ppm')
                     raise e
             else:
                 break
 
+    def _take_vm_screenshot(self, name):
+        """ Creates screenshot of VMs screeen
```
Comment (jstodola) [inline libpermian/plugins/anaconda_webui/__init__.py]: Typo in "screeen"

**Review by jstodola — DISMISSED** [review]
Looks good to me.

**Review by jstodola — APPROVED** [review]

**Pull request merged**